pom.xml

```
@@ -71,7 +71,7 @@
         <git-plugin.version>3.1.0</git-plugin.version>
         <workflow-support-plugin.version>3.3</workflow-support-plugin.version>
         <scm-api-plugin.version>2.2.6</scm-api-plugin.version>
-        <groovy-cps.version>1.28</groovy-cps.version>
+        <groovy-cps.version>1.29-20190627.174504-1</groovy-cps.version>
```

groovy-cps 1.29 was just released.
```suggestion
        <groovy-cps.version>1.29</groovy-cps.version>
```

---

FWIW in the future you accept suggestions like this directly from the GUI if desired.